frontend/src/scenes/groups/groupsListLogic.ts

```diff
@@ -155,6 +158,10 @@ export const groupsListLogic = kea<groupsListLogicType>([
             parseParam('select')
             parseParam('orderBy')
 
+            if (searchParams[`search_${props.groupTypeIndex}`] != null) {
```

Comment:
The search will not be retained when changing tabs because it seems like `groupsListLogic` can't be permanently mounted (which is what is retaining the state)

I suspect thats because `groupsListLogic` each have its own unique key, while person/cohort does not have.

https://github.com/user-attachments/assets/36817778-bbb9-459a-80cb-a2aae649d949